curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md

```diff
@@ -10,7 +10,7 @@ forumTopicId: 301047
 <section id='description'>
 Applying a color on HTML elements is not limited to one flat hue. CSS provides the ability to use color transitions, otherwise known as gradients, on elements. This is accessed through the <code>background</code> property's <code>linear-gradient()</code> function. Here is the general syntax:
 <code>background: linear-gradient(gradient_direction, color 1, color 2, color 3, ...);</code>
-The first argument specifies the direction from which color transition starts - it can be stated as a degree, where <code>90deg</code> makes a horizontal gradient (from left to right) and <code>45deg</code> is angled like a backslash. The following arguments specify the order of colors used in the gradient.
+The first argument specifies the direction from which color transition starts - it can be stated as a degree, where <code>90deg</code> makes a horizontal gradient (from left to right) and <code>45deg</code> is angled like a forward slash (from bottom left to top right). The following arguments specify the order of colors used in the gradient.
```

Comment:
I agree with adding the `(from bottom left to top right)`, but if you look at @Sky020's example, that looks like a backslash and not a forward slash, so I think the `backslash` part is more accurate.

Reply:
IMHO, I'd remove the slash reference entirely.

A 90 degree gradient creates a vertical line, with colours shifting across the horizontal axis. So this is accurate:
> where <code>90deg</code> makes a horizontal gradient (from left to right)

A 45 degree gradient creates a diagonal line `\` with colours shifting from bottom left to top right. I think saying it is angled "like a backslash" or "angled like a forward slash" both create a bit of confusion, and could be changed to mirror the 90 degree description:
> and <code>45deg</code> makes a diagonal gradient (from bottom left to top right)

Reply:
>and `45deg` makes a diagonal gradient (from bottom left to top right)

Sounds good to me.